Configuration/Views.yaml

```diff
@@ -0,0 +1,6 @@
+-
+  requestFilter: 'isPackage("Neos.Neos") && isController("Login") && isAction("index")'
```

Comment:
You also have to check for the format because the login controller provides a JsonView which would create an error if the Controller tries to set the fusionPathPatterns.